index.js

```diff
@@ -149,6 +149,10 @@ module.exports = async (input = 'patch', options) => {
 
 	tasks.add({
 		title: 'Pushing tags',
+		skip: async () => {
+			const {stdout} = await execa('git', ['status', '-sb']);
```

Comment:
Can you use the long-flags for readability?

Reply:
I think we should also use `--porcelain=2` as it's meant to make the parsing stable and easy.

Reply:
Would also be good to move Git related commands into a utility file. See: https://github.com/sindresorhus/np/pull/289#discussion_r246272308